fleet/fleet.go

```diff
@@ -217,3 +256,6 @@ func (f fleet) ipFromUnitState(unitState *schema.UnitState) (net.IP, error) {
 
 	return nil, maskAny(ipNotFoundError)
 }
+
+// TODO
```

Comment:
Would implementing `String()` on `UnitStatus` work? Then we wouldn't need this function.